### DISCUSSION TITLE
Setting sensitivity label to classic site using CSOM

### DISCUSSION THREAD

**Discussion by Vivek00001** [pnp/pnpframework#705] [Q&A]
I am trying to set sensitivity label to existing classic team site using the following CSOM code. 

  _ClientContext.Site.SensitivityLabelId = sensitivityLabelId;
  ClientContext.ExecuteQueryRetry();_

I am getting "Attempted to perform an unauthorized operation" error
I am using App-only token and App has _Sites.FullControl.All_ application permission (SharePoint).

Could you please help in identifying issue.

**Comment by LeHailender** [comment]
Any updates here?
